Java/Java.sublime-syntax

```diff
@@ -253,25 +373,33 @@ contexts:
       scope: keyword.operator.arithmetic.java
     - match: (!|&&|\|\|)
       scope: keyword.operator.logical.java
-    - match: (?<=\S)\.(?=\S)
-      scope: keyword.operator.dereference.java
+    - match: \.
+      scope: punctuation.accessor.java
```

Comment:
Additionally, `:` is used by Lua. Anyway, we can probably continue any further conversation about this on a new issue rather than flooding this PR.